packages/template-retail-react-app/app/components/reset-password/index.jsx

```diff
@@ -16,64 +16,98 @@ import ResetPasswordFields from '@salesforce/retail-react-app/app/components/for
 const ResetPasswordForm = ({submitForm, clickSignIn = noop, form}) => {
     return (
         <Fragment>
-            <Stack justify="center" align="center" spacing={8}>
-                <BrandLogo width="60px" height="auto" />
-                <Stack spacing={2}>
+            {!form.formState.isSubmitSuccessful ? (
+                <>
+                    <Stack justify="center" align="center" spacing={8}>
+                        <BrandLogo width="60px" height="auto" />
+                        <Stack spacing={2}>
+                            <Text align="center" fontSize="xl" fontWeight="semibold">
+                                <FormattedMessage
+                                    defaultMessage="Reset Password"
+                                    id="reset_password_form.title.reset_password"
+                                />
+                            </Text>
+                            <Text fontSize="sm" align="center" color="gray.700">
+                                <FormattedMessage
+                                    defaultMessage="Enter your email to receive instructions on how to reset your password"
+                                    id="reset_password_form.message.enter_your_email"
+                                />
+                            </Text>
+                        </Stack>
+                    </Stack>
+                    <form onSubmit={form.handleSubmit(submitForm)} data-testid="sf-auth-modal-form">
+                        <Stack paddingTop={8} spacing={8} paddingLeft={4} paddingRight={4}>
+                            {form.formState.errors?.global && (
+                                <Alert status="error">
+                                    <AlertIcon color="red.500" boxSize={4} />
+                                    <Text fontSize="sm" ml={3}>
+                                        {form.formState.errors.global.message}
+                                    </Text>
+                                </Alert>
+                            )}
+                            <ResetPasswordFields form={form} />
+                            <Stack spacing={6}>
+                                <Button
```

Comment:
by default, the text within is used as the aria-label so no need to explicilty set the aria-label for buttons